src/Classes/MuppetDisplay.php

```diff
@@ -17,7 +17,12 @@ public static function displayMuppets(array $muppets): string
         if (!count($muppets)) {
             return '<h1 class="error">:( All the muppets have gone out</h1>';
         }
-        $muppetString = '';
+        $muppetString = "<article><img class='randomImage' src='assets/randomMuppet.png' alt='random muppet' />"
+            . "<div><h4>Random Muppet</h4><p>1936-1996</p></div>"
+            . "<a class='button' href='details.php?muppetId=16"
+//            . $muppet>getRandomMuppet()
```

Comment:
This should be: "<a class='button' href='details.php?muppetId=" . MuppetJumble::getRandomId($muppets) . "'></a>"